src/api/error.rs

```diff
@@ -139,6 +143,8 @@ impl ApiError {
             Self::LoginRequiredForFeature(_) => "Login Required",
             Self::OpenAIError(_) => "Open AI error",
             Self::AIError(_) => "AI error",
+            Self::PaymentRequired => "Payment required",
```

Comment:
Its used in https://github.com/mdn/rumba/pull/277/files#diff-bcadba1de8298716d2007f60e8d59f39916df655a845cdfd8ac4ad4904b800edR157

Otherwise I need to return a less meaningful error or the types will get really really ugly.